src/main/java/modtrekt/ui/HelpWindow.java

```diff
@@ -15,7 +15,9 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
+    public static final String USERGUIDE_URL =
```

Comment:
Let's modify this link to use the GitHub one!
`https://ay2223s1-cs2103t-w10-4.github.io/tp/UserGuide.html`

If you want to find this, the formatting is the same as the way we obtain our IP UG/DG etc.